Program.cs

```diff
@@ -4,7 +4,7 @@
   //string hexvalue = input.ToString ("X") ----> in-built function for reference
 
 Console.Write ("Enter a decimal value: ");
-if (int.TryParse (Console.ReadLine (), out int input)) return;
+if (int.TryParse (Console.ReadLine (), out int input)) ;

```

Comment:
Remove semicolon.
Fix this error this should just print invalid input.
![image](https://github.com/RaaneshKumar/Training/assets/94427955/a796d50e-8b17-4ca6-8a27-c8a956179d76)